TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -333,7 +335,7 @@ def makeTreeFromMiniAOD(self,process):
                 subjetBTagDiscriminators = ['pfCombinedInclusiveSecondaryVertexV2BJetTags'],
                 JETCorrLevels = levels,
                 subJETCorrLevels = levels,
```

Comment:
Turning on ECF calculation for AK8: Is it okay to make this default? Or should I turn it into an option for boosted only?